ci_checks.sh

```diff
@@ -27,6 +27,7 @@ cargo check -p fuel-core-storage --target wasm32-unknown-unknown --no-default-fe
 cargo check -p fuel-core-client --target wasm32-unknown-unknown --no-default-features &&
 cargo check -p fuel-core-chain-config --target wasm32-unknown-unknown --no-default-features &&
 cargo check -p fuel-core-executor --target wasm32-unknown-unknown --no-default-features --features alloc &&
+cargo check -p fuel-core-global-merkle-root-storage --target wasm32-unknown-unknown --no-default-features &&
```

Comment:
@netrome i think green meant to add it to the `.github/workflows/ci.yml` file too :)

Reply:
Right, updated in https://github.com/FuelLabs/fuel-core/pull/2553/commits/83f9cc92e1f8f0fa53172d51f7bc90cdbadfe2c1 thanks for pointing it out :pray: